bot/seasons/halloween/scarymovie.py

```diff
@@ -18,7 +21,7 @@ class ScaryMovie:
     def __init__(self, bot):
         self.bot = bot
 
-    @commands.command(name='movie', alias=['tmdb'])
+    @commands.command(name='scarymovie', alias=['smovie'])
```

Comment:
aliases should be a tuple

Reply:
Actually, isn't the kwarg aliases?

Reply:
It is indeed aliases https://github.com/Rapptz/discord.py/blob/rewrite/discord/ext/commands/core.py#L169

Reply:
A single item in brackets will not result in a tuple. You will want a trailing comma. 
However, as a note, the aliases kwarg accepts both tuples and lists, and the type error raised explicitly mentions list in it's output:
```python
        if not isinstance(self.aliases, (list, tuple)):
            raise TypeError("Aliases of a command must be a list of strings.")
```
https://github.com/Rapptz/discord.py/blob/rewrite/discord/ext/commands/core.py#L172

Reply:
solved